src/app/zap-templates/zcl/data-model/chip/thread-network-diagnostics-cluster.xml

```diff
@@ -159,6 +164,15 @@ limitations under the License.
     <attribute side="server" code="0x3E" define="ACTIVE_THREAD_NETWORK_FAULTS" type="ARRAY" entryType="NetworkFault" length="4" writable="false" optional="false">ActiveNetworkFaultsList</attribute>
     <command source="client" code="0x00" name="ResetCounts" optional="true" cli="chip thread_network_diagnostics resetcounts">
       <description>Reception of this command SHALL reset the OverrunCount attributes to 0</description>
-    </command>           
+    </command>
+    <event side="server" code="0x00" name="ConnectionStatusEvent" priority="info" optional="true">
```

Comment:
Again, the spec does not define the priority and generally has the wrong fields in the table.  Needs spec issue.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/4643